indigo-frontend/src/core/components/common/input-suffix/input-suffix.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, forwardRef, input, output, signal } from '@angular/core';
+import {
+  ControlValueAccessor,
+  FormControl,
+  FormGroup,
+  NG_VALUE_ACCESSOR,
+  ReactiveFormsModule,
+} from '@angular/forms';
+import { SelectComponent } from '../select/select.component';
+import { InputComponent } from '../input/input.component';
+import { DropdownMenuItem } from '../dropdown-menu/dropdown-menu.i';
+import { ClassPickerPipe } from '../../../pipes/classPicker.pipe';
+import { InputSuffixValue } from '../../../types/input-suffix.i';
+
+@Component({
+  selector: 'app-input-suffix',
+  templateUrl: './input-suffix.component.html',
+  standalone: true,
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => InputSuffixComponent),
+      multi: true,
+    },
+  ],
+  imports: [
+    SelectComponent,
+    InputComponent,
+    ClassPickerPipe,
+    ReactiveFormsModule,
+  ],
+})
+export class InputSuffixComponent implements ControlValueAccessor {
+  label = input.required<string>();
+  placeholder = input<[string, string]>();
```

Comment:
Sorry, forgot to add the same comment as for value. I think we can use object here too.